<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Omron Tag Converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icons/dev-32/dev-32.ico</normaloff>:/icons/dev-32/dev-32.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>651</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QPlainTextEdit" name="inputTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>631</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="proceedButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>222</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Конвертировать</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>651</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QPlainTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>631</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="savePath">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>269</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
